app/controllers/v0/letters_discrepancy_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# imports needed for evss
+require 'common/exceptions/record_not_found'
+require 'evss/letters/download_service'
+require 'evss/letters/service'
+
+# imports needed for lighthouse
+require 'lighthouse/letters_generator/service'
+require 'lighthouse/letters_generator/service_error'
+require 'lighthouse/letters_generator/veteran_sponsor_resolver'
+
+module V0
+  class LettersDiscrepancyController < ApplicationController
+    service_tag 'letters'
+    before_action { authorize :evss, :access_letters? }
+    before_action { authorize :lighthouse, :access? }
+
+    def index
+      # Call lighthouse endpoint
+      lh_response = lh_service.get_eligible_letter_types(@current_user.icn)
+      # Format response into lh_letters array
+      lh_letters = lh_response[:letters].pluck(:letterType)
+
+      # Call evss endpoint
+      evss_response = evss_service.get_letters
+      # Format response into evss_letters array
+      evss_letters = evss_response.letters.map(&:letter_type)
+
+      # Return if there are no differences
+      render json: [] and return if lh_letters.sort == evss_letters.sort
```

Comment:
I would make sure you are returning the same data here and on L46 i.e. either: `[]` or `{ data : [] }`

Reply:
Though if you are going to render anyways, I don't see why you don't just do something like:
```ruby
unless lh_letters.sort == evss_letters.sort
  # log differences
end

render json: []
```

Reply:
> I would make sure you are returning the same data here and on L46 i.e. either: `[]` or `{ data : [] }`

@jerekshoe  This actually was our attempt to make them output the same thing consistently, because the other one returned `{ data: [] }` to the browser. But to your follow up point, that should make this moot once we reduce to a single render. cc: @pmclaren19